x-pack/test/functional/apps/security/feature_privileges/dashboard.js

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from 'expect.js';
+// eslint-disable-next-line max-len
+import { DashboardConstants, createDashboardEditUrl } from '../../../../../../src/legacy/core_plugins/kibana/public/dashboard/dashboard_constants';
+
+export default function ({ getPageObjects, getService }) {
+  const esArchiver = getService('esArchiver');
+  const security = getService('security');
+  const kibanaServer = getService('kibanaServer');
+  const PageObjects = getPageObjects(['common', 'dashboard', 'security', 'spaceSelector']);
+  const testSubjects = getService('testSubjects');
+
+  describe('dashboard', () => {
+    before(async () => {
+      await esArchiver.load('security/feature_privileges');
+      await kibanaServer.uiSettings.replace({
+        "accessibility:disableAnimations": true,
+        "telemetry:optIn": false,
+        "defaultIndex": "logstash-*",
+      });
+      await esArchiver.loadIfNeeded('logstash_functional');
```

Comment:
For my own education, why is this one loaded only if needed? Why doesn't it have to be unloaded?

Reply:
Honestly, I was following the pattern that we've followed elsewhere. The logstash data isn't modified, so reloading it between tests/suites isn't really required and it's somewhat costly operation, so it can be faster to just leave it there when you're done incase anyone needs it. If they need other data for the same indices, it should blow them away.